### DISCUSSION TITLE
[Bug report] JavaScript error occurred in the main process

### DISCUSSION THREAD

**Discussion by aoedk** [GetPublii/Publii#1867] [Bug Report]
### Operating system

macOS

### Publii version

0.46.2

### Issue type

Application

### Bug description

When starting Publli with a shared folder via Dropbox, this error occurs.

<img width="519" alt="publii-js-error" src="https://github.com/user-attachments/assets/27ba33d8-7e9f-43f1-b868-9d2c7b330b76">

#1643 looks related, but this is an Apple MacBook, not during installation and it seems related to Dropbox.

### Steps to reproduce

Three editors sharing one or more sites via Dropbox. One editor (getting the error) might have moved the shared folder in Dropbox, but moved it back and reinstalled Publii.

**Comment by michaelfluegge** [comment]
That is no bug in Publii. That is an issue of starting Publii before syncing via Dropbox was completed or similar.

**Reply by philipptemmel** [reply]
I am having the very same issue whenever I try syncing the website. Nothing happens or progresses, but when I cancel I get the same error message. 

So, if it is related to Dropbox, any ideas how to fix it? My Dropbox sync is done, but Publii is not working, even after reinstalling everything.

**Reply by michaelfluegge** [reply]
Which files/folders do you sync via dropbox?
